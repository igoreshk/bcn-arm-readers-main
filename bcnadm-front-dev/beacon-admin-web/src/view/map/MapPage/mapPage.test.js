import React from 'react';
import { shallow } from 'enzyme';
import { MapPage } from './index';

const props = {
  selectedMapProviderName: 'Google',
  fetchLevel: jest.fn(),
  match: {
    params: { level: '5beea5a30bd42e1fb05bc0ea' }
  }
};

const wrapper = shallow(<MapPage {...props} />);

describe('Map', () => {
  it('Check matching snapshot', () => expect(wrapper).toMatchSnapshot());
});
