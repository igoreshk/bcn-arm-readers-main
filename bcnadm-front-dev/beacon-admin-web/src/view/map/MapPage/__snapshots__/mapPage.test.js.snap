// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Check matching snapshot 1`] = `
<div
  className="mapPage"
>
  <withRouter(LocalizedComponent) />
  <Connect(MapRouter)
    buildingCenterCoordinates={Array []}
    loadedOSMap={
      Object {
        "current": undefined,
      }
    }
    picture={Array []}
    pictureCoords={Array []}
    selectedMapProviderName="Google"
  />
  <Suspense
    fallback={<LoadingScreen />}
  >
    <lazy
      buildingCenterCoordinates={Array []}
      loadedOSMap={
        Object {
          "current": undefined,
        }
      }
      picture={Array []}
      pictureCoords={Array []}
      selectedMapProviderName="Google"
      setBuildingCenterCoordinates={[Function]}
      setPicture={[Function]}
      setPictureCoords={[Function]}
      shapes={Array []}
    />
  </Suspense>
</div>
`;
